__tests__/src/action.test.ts

```diff
@@ -289,6 +289,24 @@ describe('Given an Action', () => {
       }
     })
 
+    it('When the filename for an artifact file has more than 255 chars then it is limited', async () => {
+      let randomChars = 'asd1234'
+      const executionGraph = executionGraphMother.empty(undefined, undefined, [ taskMother.trivy() ])
+      const executionGraphReport = executionGraphReportMother.report()
+      jest.spyOn(action.vib, 'getRawLogs').mockResolvedValue('test raw logs')
+      jest.spyOn(action.vib, 'getRawReports').mockResolvedValue([{id: 'test-id', mime_type: 'text/html', filename: `${randomChars += (Math.floor(Math.random() * 60))}`.slice(0, 255)}])
```

Comment:
Actually, I'm thinking it may be more clear if we just use the filename from the example mentioned in the original issue:

```suggestion
    it('When the filename for an artifact file has more than 255 chars then it is limited', async () => {
      const filename = '6e1bd432-b159-4ee1-bc72-42e69b775a8d_vmwaresaas-jfrog-io-content-platform-docker-containers-'
      + 'modern-spring-on-kubernetes-buildpacks-fc4924b55b73814cacc1f2727d33587bb1525841-1668544950160-sha256-37fd181'
      + '6bfdcaf2ec873b89789261baa668a9efa831499d249fb9a816536252b.json'
      const executionGraph = executionGraphMother.empty(undefined, undefined, [ taskMother.trivy() ])
      const executionGraphReport = executionGraphReportMother.report()
      jest.spyOn(action.vib, 'getRawLogs').mockResolvedValue('test raw logs')
      jest.spyOn(action.vib, 'getRawReports').mockResolvedValue([{id: 'test-id', mime_type: 'text/html', filename: filename}])
```